CHANGELOG.md

```diff
@@ -3,7 +3,7 @@
 ## main / unreleased
 
 ### Grafana Mimir
-
+* [CHANGE] Ingester: hte configuration parameter `-ingester.ring.readiness-check-ring-health` has been deprecated and will be removed in Mimir 2.7. #4422
```

Comment:
Mimir 2.7 release is now in progress. If this PR is merged soon and is part of Mimir 2.7, we can say that option will be removed in Mimir 2.9.